extensions/wikia/ARecoveryEngine/ARecoveryEngine.i18n.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+$messages = [ ];
+
+$messages["en"] = [
+	"arecovery-blocked-message-headline" => "Ad blocker interference detected!",
+	"arecovery-blocked-message-part-one" => "Wikia is a free-to-use site that makes money from advertising. We have a modified experience for viewers using ad blockers",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "arecovery-blocked-message-part-one" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Wikia is a free-to-use site that makes money from advertising. We have a modified experience for viewers using ad blockers" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)